bio/jellyfish/dump/environment.yaml

```
@@ -0,0 +1,6 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - kmer-jellyfish==2.3.0
```

As above, I'd suggest:
```suggestion
  - kmer-jellyfish==2.3
```